fusor-ember-cli/app/styles/wizard.scss

```diff
@@ -49,10 +49,13 @@
   width: 50%;
 }
 .rhci-steps li.active {
-  color: #0099d3;
+  color: #0099d3 !important;
```

Comment:
For some reason, using the `ember-cli-active-link-wrapper` required me to add `!important` or the active links were not blue.